src/main/resources/i18n/strings.properties

```
@@ -147,29 +147,38 @@ migration.impossible.reason=The vault cannot be automatically migrated because i
 migration.impossible.moreInfo=The vault can still be opened with an older version. For instructions on how to manually migrate a vault, visit
 
 # Health Check
-health.title=Vault Health Check
-health.start.configValid=Reading and parsing vault configuration file was successful. Proceed to select checks.
-health.start.configInvalid=Error while reading and parsing the vault configuration file.
-health.checkList.header=Available Health Checks
-health.checkList.selectAllBox=Select All
+## Start
+health.title=Health Check of "%s"
+health.intro.header=Health Check
+health.intro.text=Health Check is a collection of checks to detect and possibly fix problems in the internal structure of your vault. Please keep in mind:
+health.intro.remarkSync=Ensure all devices are completely synced, this resolves most problems.
+health.intro.remarkFix=Not all problems can be fixed.
+health.intro.remarkBackup=If data is corrupted, only a backup can help.
+health.intro.affirmation=I have read and understood the above information
+## Start Failure
+health.fail.header=Error on loading Vault Configuration
+health.fail.ioError=An error happened while accessing and reading the config file : "%s" 
```

```suggestion
health.fail.ioError=An error happened while accessing and reading the config file.
```

Equivalent to `health.fail.parseError`. If you'd like to show more information (`%s`), make that in code.